src/flows/context/flow.query.tsx

```diff
@@ -393,7 +403,13 @@ export const FlowQueryProvider: FC = ({children}) => {
   }
 
   const simple = (text: string) => {
-    return simplify(text, vars)
+    const override = {
+      vars,
+    } as QueryScope
+
+    return isFlagEnabled('GlobalQueryContext')
+      ? getQuery(text, override).simplify()
```

Comment:
wait. i forget the end of the conversation about overrides being embedded in the query, or external to the query. is this what we talked about or something missed?